### PR TITLE
WIP

### PR DESCRIPTION
**Pull request by mel-d**

### CODE REVIEW

**Comment by mel-d** [comment]
I think I have my string segmenter included with files in the right place--what I have been struggling with is how to get the user's url input to be used as the string. I made a version of my segmenter class called Website, and when I run that through my segmentation program, I gave it the variable @please_work. I added a parameter to the 'running through segmentation' method in an attempt to at least head in the right direction, but I feel kinda lost as to how I can get the user input plugged into that.

**Comment by sumeetjain** [comment]
How did you access the value of the user's URL input in the simple web application? (Go check the actual code.)

File: controllers/main_controller.rb [hunk 1]
```diff
@@ -9,3 +11,15 @@
   # combined document back to the client.
   erb :"main/welcome"
 end
+
+MyApp.get "/:num" do
+  Website = Segmentation.new
```
Comment (sumeetjain) [inline controllers/main_controller.rb]: In Ruby, don't capitalize variables. Capitalized keywords are classes. So this should be `website = Segmentation.new`.

**Comment by mel-d** [comment]
I've added the method we used yesterday for getting user's input--but I feel like I've tried it in every possible place within my code, and no matter where it is I keep getting a message that my input is nil.

**Comment by mel-d** [comment]
I fixed the error you had found in my code. I'm still getting the issue 

NoMethodError at /turtleblue
undefined method `run_through_segmentation' for nil:NilClass

```
file: segment_page.erb location: block in singleton class line: 5 
```

when I try to load the webpage after rackup.

**Pull request closed without merging**